config/r9k/README.md

```diff
@@ -0,0 +1 @@
+The checksums used by Robot9000 mode are stored here, one file per room.
```

Comment:
I would suggest that in general, especially when yours uses such large amounts of data.